php/class-wp-cli.php

```diff
@@ -819,8 +819,23 @@ public static function error_multi_line( $message_lines ) {
 	 * @param array $assoc_args Skips prompt if 'yes' is provided.
 	 */
 	public static function confirm( $question, $assoc_args = array() ) {
+		$args = wp_parse_args( $assoc_args, array(
```

Comment:
`wp_parse_args()` isn't necessarily available yet. You'll want to use `array_merge()` instead.

Reply:
Ah, right, my bad. WordPress may or may not be loaded yet.